backend/src/main/java/ai/verta/modeldb/reconcilers/ReconcilerInitializer.java

```diff
@@ -25,32 +26,34 @@
   public static void initialize(
       Config config, ServiceSet services, DAOSet daos, FutureJdbi futureJdbi, Executor executor) {
     LOGGER.info("Enter in ReconcilerUtils: initialize()");
+
+    ReconcilerConfig reconcilerConfig = new ReconcilerConfig();
+    reconcilerConfig.setTestReconciler(config instanceof TestConfig);
```

Comment:
This initialization and then setting extra things is an anti-pattern. We should handle everything in the initialization.

Reply:
Fixed and used the constructor to initialize field.